swap/src/cli/command.rs

```diff
@@ -7,16 +7,29 @@ use std::str::FromStr;
 use url::Url;
 use uuid::Uuid;
 
-pub const DEFAULT_STAGENET_MONERO_DAEMON_ADDRESS: &str = "monero-stagenet.exan.tech:38081";
+// See: https://moneroworld.com/
+pub const DEFAULT_STAGENET_MONERO_DAEMON_ADDRESS: &str = "node.moneroworld.com:18089";
+pub const DEFAULT_STAGENET_MONERO_DAEMON_ADDRESS_TESTNET: &str = "monero-stagenet.exan.tech:38081";
```

Comment:
The naming seems to be slightly off here :sweat_smile: 

```suggestion
pub const DEFAULT_MAINNET_MONERO_DAEMON_ADDRESS: &str = "node.moneroworld.com:18089";
pub const DEFAULT_STAGENET_MONERO_DAEMON_ADDRESS: &str = "monero-stagenet.exan.tech:38081";
```